logs/app.log

```diff
@@ -0,0 +1,120 @@
+[2023-10-07 13:03:56,893] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed image shape: torch.Size([1, 1, 512, 512, 111])
+[2023-10-07 13:03:56,899] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed label shape: torch.Size([1, 1, 512, 512, 111])
+[2023-10-07 13:03:56,902] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed pred shape: torch.Size([1, 1, 512, 512, 111])
+[2023-10-07 13:05:40,687] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed image shape: torch.Size([1, 1, 512, 512, 410])
+[2023-10-07 13:05:40,692] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed label shape: torch.Size([1, 1, 512, 512, 410])
+[2023-10-07 13:05:40,696] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed pred shape: torch.Size([1, 1, 512, 512, 410])
+[2023-10-07 13:07:41,407] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed image shape: torch.Size([1, 1, 512, 512, 463])
+[2023-10-07 13:07:41,413] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed label shape: torch.Size([1, 1, 512, 512, 463])
+[2023-10-07 13:07:41,417] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed pred shape: torch.Size([1, 1, 512, 512, 463])
+[2023-10-07 13:07:41,511] [DEBUG] [liver_imaging_analysis.models.liver_segmentation] (liver_segmentation.py:755) - Initial test loss=0.9389247417449951
+[2023-10-07 13:07:41,518] [DEBUG] [liver_imaging_analysis.models.liver_segmentation] (liver_segmentation.py:760) - Initial test metric=0.05373082309961319
+[2023-10-07 13:07:41,522] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:409) - 
+Epoch 1/100
+-------------------------------
+[2023-10-07 13:07:41,527] [INFO] [liver_imaging_analysis.engine.utils] (utils.py:122) - 
+|____________________________________________________________________________________________________|  0.00%  ---> 0/111
+[2023-10-07 13:08:02,996] [INFO] [liver_imaging_analysis.engine.utils] (utils.py:122) - 
+|____________________________________________________________________________________________________|  0.90%  ---> 1/111
+[2023-10-07 13:08:03,717] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed image shape: torch.Size([2, 1, 96, 96, 64])
+[2023-10-07 13:08:03,722] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed label shape: torch.Size([2, 1, 96, 96, 64])
+[2023-10-07 13:08:03,725] [DEBUG] [liver_imaging_analysis.engine.engine] (engine.py:195) - Post processed pred shape: torch.Size([2, 1, 96, 96, 64])
+[2023-10-07 13:08:55,573] [INFO] [liver_imaging_analysis.engine.utils] (utils.py:122) - 
+|#___________________________________________________________________________________________________|  1.80%  ---> 2/111
```

Comment:
Repeating these lines for every batch will make the file too large that you will not be able to extract important information from it. Maybe, you can log it during the first batch, and when the values change.

Reply:
done